docs/page/css/syntax.css

```diff
@@ -77,3 +77,6 @@
 .highlight .vg { color: #008080 } /* Name.Variable.Global */
 .highlight .vi { color: #008080 } /* Name.Variable.Instance */
 .highlight .il { color: #009999 } /* Literal.Number.Integer.Long */
+
+/* Missing YAML styles */
+.highlight .l-Scalar-Plain { color: #008080 }
```

Comment:
I think we should merge this PR without this change and look at fixing this in another ticket. There are [other](https://highlightjs.org) highlighters with true YAML support.